aiounifi/models/device.py

```diff
@@ -330,6 +330,19 @@ class TypedDeviceUplink(TypedDict):
     uplink_remote_port: int
 
 
+class TypedDeviceUpdtimeStats(TypedDict):
+    """Device uptime stats type definition."""
+
+    WAN: TypedDeviceUpdtimeStatsWan
+    WAN2: TypedDeviceUpdtimeStatsWan
+
+
+class TypedDeviceUpdtimeStatsWan(TypedDict):
+    """Device uptime stats wan type definition."""
+
+    monitors: list[dict[str, Any]]
```

Comment:
Please make a class that defines the attributes

{
              "availability": 100.0,
              "latency_average": 5,
              "target": "www.microsoft.com",
              "type": "icmp"
            },